static/src/router/index.js

```diff
@@ -1,47 +1,39 @@
-import Vue from 'vue';
+import Vue from 'vue'
 import Router from 'vue-router';
 
-Vue.use(Router);
+Vue.use(Router)
 
 const router = new Router({
   mode: 'history',
   routes: [{
-    path: '',
-    redirect: '/login',
-  },
-  {
-    path: '/login',
-    name: 'Auth',
-    component: () => import('../views/Authentication.vue'),
-  },
-  {
-    path: '/chat',
-    name: 'chat',
-    component: () => import('../views/Chat.vue'),
-    meta: {
-      auth: true,
+      path: '',
+      redirect: '/login',
     },
-  },
-  {
-    path: '/bookmarks',
-    name: 'bookmarks',
-    component: () => import('../views/Bookmarks.vue'),
-    meta: {
-      auth: true,
+    {
+      path: '/login',
+      name: 'Auth',
+      component: () => import('../views/Authentication.vue')
     },
-  },
-  ],
-});
+    {
+      path: '/chat',
+      name: 'chat',
+      component: () => import('../views/Chat.vue'),
+      meta: {
+        auth: true
+      }
+    }
+  ]
+})
 
 const token = localStorage.getItem('Token');
 router.beforeEach((to, from, next) => {
-  if (to.name === 'login') {
+  if (to.name == 'login') {
```

Comment:
Curious why "==" here instead of "===". I tend to use "===" for nearly everything, so if there are more instances where I should rather be using "==", I'd be keen to learn when & why.